### PR TITLE
[AssetMapper] Minor: Reword warning message

### PR DESCRIPTION
**Pull request by ThomasLandauer**
| Q             | A
| ------------- | ---
| Branch?       | 7.1
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Issues        | 
| License       | MIT

"in development" is somewhat unclear.
But since the criteria is `if ($this->isDebug)`, the new "in dev enviroment" isn't completely true either, so maybe rephrase it to something like "in an environment with debug mode enabled"?

### CODE REVIEW

**Comment by yceruto** [comment]
I'd go with the 2nd proposal, which is more accurate:
```
"in an environment with debug mode enabled"
```

**Comment by ThomasLandauer** [comment]
OK, I changed it.

File: src/Symfony/Component/AssetMapper/Command/AssetMapperCompileCommand.php [hunk 1]
```diff
@@ -89,7 +89,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
         if ($this->isDebug) {
             $io->warning(sprintf(
-                'You are compiling assets in development. Symfony will not serve any changed assets until you delete the files in the "%s" directory.',
+                'You are compiling assets in an environment with debug mode enabled. Symfony will not serve any changed assets until you delete the files in the "%s" directory again.',
```
Comment (nicolas-grekas) [inline src/Symfony/Component/AssetMapper/Command/AssetMapperCompileCommand.php]:
```suggestion
                'You are compiling assets with debug mode enabled. Symfony will not serve any changed assets until you delete the files in the "%s" directory again.',
```

**Review by nicolas-grekas — APPROVED** [review]

File: src/Symfony/Component/AssetMapper/Command/AssetMapperCompileCommand.php (hunk 1 above)
Comment (ThomasLandauer) [reply src/Symfony/Component/AssetMapper/Command/AssetMapperCompileCommand.php]: This sounds like debug mode is a parameter of `asset-map:compile`. I changed it again - what do you think?

File: src/Symfony/Component/AssetMapper/Command/AssetMapperCompileCommand.php [hunk 2]
```diff
@@ -89,7 +89,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
         if ($this->isDebug) {
             $io->warning(sprintf(
-                'You are compiling assets in development. Symfony will not serve any changed assets until you delete the files in the "%s" directory.',
+                'Debug mode is enabled in your project. Therefore Symfony will not serve any changed assets until you delete the files in the "%s" directory again.',
```
Comment (nicolas-grekas) [inline src/Symfony/Component/AssetMapper/Command/AssetMapperCompileCommand.php]: ?
```suggestion
                'Debug mode is enabled in your project: Symfony will not serve any changed assets until you delete the files in the "%s" directory again.',
```

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @ThomasLandauer.
